src/SWP/Bundle/BridgeBundle/Resources/config/doctrine-orm/Item.orm.yml

```diff
@@ -0,0 +1,54 @@
+SWP\Component\Bridge\Model\Item:
```

Comment:
why do you need to keep item in a separate table? can't it be in one table (swp_package) as a serialized array collection? Serializer would then automatically handle it, and you won't need to fetch data based on the relation.